pkg/mountmanager/mount_linux.go

```diff
@@ -0,0 +1,61 @@
+// +build linux
+
+/**
+ * Copyright 2021 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package mountmanager ...
+package mountmanager
+
+import (
+	"os"
+
+	mount "k8s.io/mount-utils"
+)
+
+// MakeFile creates an empty file.
+func (m *NodeMounter) MakeFile(path string) error {
```

Comment:
@sameshai Earlier in k8s v1.14, these functions were part of the mount package, https://github.com/kubernetes/kubernetes/blob/release-1.14/pkg/util/mount/mount_linux.go#L389-L412

Reply:
Ok got it